src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  let tabContent = tabs[0].content;
+  let displayTabId = activeTabId;
+
+  if (!tabs.find(tab => tab.id === activeTabId)) {
+    displayTabId = tabs[0].id;
+  }
```

Comment:
from the 4th to the 9th line, you can replace everything with one function like that and use it in the code
const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];